vulnerable_people_form/form_pages/shared/session.py

```diff
@@ -293,6 +295,7 @@ def load_answers_into_session_if_available():
             "basic_care_needs": do_you_need_help_meeting_your_basic_care_needs["longValue"],
             "do_you_have_someone_to_go_shopping_for_you": do_you_have_someone_to_go_shopping_for_you["longValue"],
             "do_you_live_in_england": do_you_live_in_england.get("longValue"),
+            "tier_at_submission": tier_at_submission["longValue"],
```

Comment:
this should be get as the value can be null

Reply:
Ahhh, you're right